backend.native/tests/interop/j2objc/Foo.java

```diff
@@ -0,0 +1,14 @@
+public class Foo {
```

Comment:
I think something needs to be changed in a build rule to force this to be compilied into a jar file too.

Reply:
Yes, I agree. I kept it like this for now since I never ended up finding where the tests build-rules were originally. This is something that needs to be done though.